src/dash-table/components/ControlledTable/index.tsx

```diff
@@ -718,7 +700,10 @@ export default class ControlledTable extends PureComponent<ControlledTableProps>
         const { r0c1 } = this.refs as { [key: string]: HTMLElement };
 
         Logger.trace(`ControlledTable fragment scrolled to (left,top)=(${ev.target.scrollLeft},${ev.target.scrollTop})`);
-        r0c1.style.marginLeft = `${-ev.target.scrollLeft}px`;
+
+        const margin = parseFloat(ev.target.scrollLeft) + parseFloat(getComputedStyle(r0c1).marginRight);
```

Comment:
`margin-left` and `margin-right` match in dimension, if not sign. Scroll adds to the initial offset.